fakeserver/routes.go

```diff
@@ -257,6 +261,35 @@ func postV1AssignmentOverride(r *http.Request) error {
 	return nil
 }
 
+func postV2AssignmentOverride(r *http.Request) error {
+	var assignments []v1Assignment
+	contentType := r.Header.Get("content-type")
+	switch {
+	case strings.HasPrefix(contentType, "application/json"):
+		requestBytes, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			return err
+		}
+		var assignmentMap map[string][]v1Assignment
+		err = json.Unmarshal(requestBytes, &assignmentMap)
+		if err != nil {
+			return err
+		}
+		assignments = assignmentMap["assignments"]
```

Comment:
keeping things simple. the request looks like this:

```json
{
  "assignments":[
    {"split_name": "allocation_selection_refactor_q1_2018_enabled", "variant": "true"}, 
    {"split_name":"add_preferred_muni_bond_locale_selection_to_dr_enabled", "variant":"false"}
  ]
}
```
so, i'm just unmarshaling that as a map of string to v1Assignment and then pulling the value for that one key